examples/OECC/README.md

```diff
@@ -0,0 +1,134 @@
+# OpenEdge Command Center #
+
+Demo scripts to setup an OpenEdge Command Center environment with support for OpenTelemetry metrics.
+
+The scripts can be run on a virtual machine running Ubuntu Linux 22.04.
+The VM could be created on AWS, Azure or other virtualization environment.
+Scripts run commands to detect the IP address of the machine.
+
+### Requirements 
+* Ubuntu Linux 22.04
+    * user with sudo access
+* OpenEdge media:
+    * PROGRESS_OE_12.8.3_LNX_64.tar.gz
+    * PROGRESS_OECC_SERVER_1.3.0_LNX_64.tar.gz
+    * PROGRESS_OECC_AGENT_1.3.0_LNX_64.bin
+* Reponse file for OpenEdge installation:
+    * response_12.8.3.ini
+    * Products:
+        * OE Adv. Ent. RDBMS
+        * 4GL Development System
+        * Progress ProdAS for OE
+
+**Notes:**
+* You can use the OpenEdge-Samples/examples/OECC/config/response_12.8.3.ini file as a template and specify the control codes in the file.
```

Comment:
When calling out files or commands inline with text, try to use `backticks` to visually isolate elements

Reply:
Added backticks.